project/EndpointsSettings.scala

```diff
@@ -99,7 +98,7 @@ object EndpointsSettings {
   // --- Common dependencies
 
   val circeVersion = "0.13.0"
-  val playjsonVersion = "2.8.1"
+  val playjsonVersion = "2.9.0"
```

Comment:
Is it compatible with Play 2.8.2?

Reply:
I think it is, also only `2.9.0` has published artifacts for scala.js 1.1.0 https://mvnrepository.com/artifact/com.typesafe.play/play-json